wire/msggetaddrv2.go

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) 2013-2015 The btcsuite developers
+// Copyright (c) 2015-2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package wire
+
+import (
+	"fmt"
+	"io"
+)
+
+// MsgGetAddrV2 implements the Message interface and represents a decred
+// getaddr message.  It is used to request a list of known active peers on the
+// network from a peer to help identify potential nodes.  The list is returned
+// via one or more addrv2 messages (MsgAddrV2).
+//
+// This message has no payload.
+type MsgGetAddrV2 struct{}
```

Comment:
I commented as I was reviewing, but after having gone through the entire commit, I don't really see any reason to introduce `getaddrv2` given it is identical to `getaddr` and the protocol version determines whether or not to respond with a `addr` or `addrv2`.